assets/src/blocks/TakeActionBoxout/TakeActionBoxoutEditor.js

```diff
@@ -36,6 +36,9 @@ export const TakeActionBoxoutEditor = ({
     stickyOnMobile,
   } = attributes;
 
+  const urlParams = new URLSearchParams(window.location.search);
+  const postId = urlParams.get('post');
```

Comment:
Maybe [this](https://github.com/greenpeace/planet4-master-theme/blob/main/assets/src/blocks/Articles/ArticlesEditor.js#L148) is a better way to get the post id?